commands/azureCommands/pull-from-azure.ts

```diff
@@ -22,11 +25,13 @@ export async function pullFromAzure(context?: AzureImageTagNode | AzureRepositor
         registryName = context.registry.loginServer;
         registry = context.registry;
 
-        if (context.contextValue === "azureImageTagNode") { // Right Click on AzureImageNode
+        if (context instanceof AzureImageTagNode) { // Right Click on AzureImageNode
             imageName = context.label;
-        } else { // Right Click on AzureRepositoryNode
+        } else if (context instanceof AzureRepositoryNode) { // Right Click on AzureRepositoryNode
             console.log(context.repositoryName);
             imageName = `${context.label} -a`; // Pull all images in repository
+        } else {
+            assert.fail(`Unexpected node type: ${context}`);
```

Comment:
That converts to this string: "Unexpected node type: [object Object]".  Use ${context.contextValue} to give us something useful.

Reply:
Tslint says "Property not found on type never". It shouldn't get to this else statement since the parameters are either AzureImageTagNode or AzureRepositoryNode. I'll just remove the variable all together